radio/src/lua/CMakeLists.txt

```diff
@@ -1,14 +1,23 @@
 
+if(WIN32)
+  set(RM del)
+  find_program(GREP grep HINTS "c:/cygwin/bin/" "c:/msys/bin/" "c:/GnuWin32/bin/" DOC "GNU grep utility (eg. c:/cygwin/bin/grep.exe).")
+  if(NOT GREP)
+    message(WARNING "GNU grep not found, Lua exports cannot be generated.")
+  endif()
+else()
+  set(RM rm)
+  set(GREP grep)
+endif()
+
 macro(add_lua_export_target target)
-  if(PYTHONINTERP_FOUND)
-    add_custom_command(
-      OUTPUT lua_exports_${target}.inc
-      COMMAND ${CMAKE_C_COMPILER} -E ${ARGN} -DEXPORT ${RADIO_SRC_DIRECTORY}/dataconstants.h > lua_exports_${target}.tmp
-      COMMAND grep LEXP lua_exports_${target}.tmp > lua_exports_${target}.txt
-      COMMAND rm lua_exports_${target}.tmp
-      COMMAND ${PYTHON_EXECUTABLE} ${RADIO_DIRECTORY}/util/luaexport.py ${VERSION} lua_exports_${target}.txt lua_exports_${target}.inc lua_fields_${target}.txt
+  add_custom_command(
+    OUTPUT lua_exports_${target}.inc
+    COMMAND ${CMAKE_C_COMPILER} -E ${ARGN} -DEXPORT ${RADIO_SRC_DIRECTORY}/dataconstants.h > lua_exports_${target}.tmp
+    COMMAND ${GREP} LEXP lua_exports_${target}.tmp > lua_exports_${target}.txt
+    COMMAND ${RM} lua_exports_${target}.tmp
+    COMMAND ${PYTHON_EXECUTABLE} ${RADIO_DIRECTORY}/util/luaexport.py ${VERSION} lua_exports_${target}.txt lua_exports_${target}.inc lua_fields_${target}.txt
```

Comment:
We could even remove the grep from the equation, the script `luaexport.py` would then have to do the filtering itself(should be very simple to implement).